tests/cstor/gtest/gtest_utils.cc

```diff
@@ -64,7 +64,14 @@ std::string GtestUtils::getCmdPath(std::string zfsCmd) {
 	std::string cmdPath;
 	const char *srcPath = std::getenv("SRC_PATH");
 
-	if (srcPath == nullptr) {
+	// Since the zrepl has been moved from cstor
+	// to libcstor base path of zrepl will be ../libcstor
```

Comment:
we should probably move the test also to the libcstor.

Reply:
Can I do it in separate PR?

Reply:
surr @sai.

Reply:
Thanks